src/modules/cpu/kernel/flip.hpp

```diff
@@ -88,7 +88,7 @@ RppStatus flip_u8_u8_host_tensor(Rpp8u *srcPtr,
 
         //Compute constant increment, Decrement factors used in source pointer updation
         Rpp32s srcPtrIncrement = (horizontalFlag)? -vectorIncrement : vectorIncrement;
-        Rpp32u hFlipFactor = (vectorIncrement - 1) * horizontalFlag;
+        Rpp32u hFlipFactor = (vectorIncrement - 3) * horizontalFlag;
```

Comment:
any reason for -3 - add a comment if non-intuitive reasoning

Reply:
it is incase of pkd3 . we need move to three channel and load in reverse order for flip